woocommerce/woocommerce
# Update Playwright to 1.44.1 from 1.41.1

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





We had fallen a bit behind in Playwright in the monorepo, mostly because Playwright introduced [this breaking change](https://github.com/microsoft/playwright/issues/29734) a few versions back.  This PR updates Playwright to the latest release and refactors some shopper/merchant tests in core so that they're compatible with the latest version.

Also updates the Blocks e2e tests to the latest version (ran tests, and no apparent issues with new version of Playwright)



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure CI passes for both Core and Blocks e2e tests
2. `pnpm env:restart && pnpm test:e2e-pw` (for core tests).  Will probably be prompted to install updated browsers first.
3. `pnpm env:restart && pnpm test:e2e` (for blocks tests)



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
